packages/venia-ui/lib/components/WishlistPage/wishlist.js

```diff
@@ -28,14 +28,18 @@ const Wishlist = props => {
     const contentClass = isOpen ? classes.content : classes.content_hidden;
     const contentToggleIconSrc = isOpen ? ChevronUp : ChevronDown;
     const contentToggleIcon = <Icon src={contentToggleIconSrc} size={24} />;
-    const visibilityLabel = sharingCode
-        ? formatMessage({ id: 'wishlist.publicText', defaultMessage: 'Public' })
-        : formatMessage({
-              id: 'wishlist.privateText',
-              defaultMessage: 'Private'
-          });
+    const visibilityLabel =
+        visibility === 'PUBLIC'
+            ? formatMessage({
+                  id: 'wishlist.publicText',
+                  defaultMessage: 'Public'
+              })
+            : formatMessage({
+                  id: 'wishlist.privateText',
+                  defaultMessage: 'Private'
+              });
```

Comment:
Should we use the globals here instead?

```suggestion
    const visibilityLabel =
        visibility === 'PUBLIC'
            ? formatMessage({
                  id: 'global.public',
                  defaultMessage: 'Public'
              })
            : formatMessage({
                  id: 'global.private',
                  defaultMessage: 'Private'
              });
```

Reply:
Yes it makes sense for this phrase "global", and these global phrases already used in other wishlist component)
Fixed.
Thank you